src/components/Cart.jsx

```diff
@@ -0,0 +1,156 @@
+import React from "react";
+import { useProducts } from "../context/product-context";
+import { getTotal } from "../context/utils/filter";
+import Header from "./Header";
+import Rating from "./Rating";
+
+const CartCard = ({
+  id,
+  category,
+  image,
+  title,
+  description,
+  price,
+  rating,
+}) => {
+  const { state, dispatch } = useProducts();
```

Comment:
suggestion: if you could destructure the state here you can easily use varaible name instead of doin doing state.cart